DigitalLearningSolutions.Web/Helpers/SearchHelper.cs

```diff
@@ -1,163 +1,107 @@
 ﻿namespace DigitalLearningSolutions.Web.Helpers

```

Comment:
I'm assuming this is just formatted and not changed? :)

Reply:
If so, could we add a comment at the top of this saying that we're trying to move away from the non-generic version, pointing the user to the new file? 🙏

Reply:
I've also removed the unused methods for courses + frameworks. Same for SortingHelper